kolibri/deployment/default/settings/base.py

```diff
@@ -331,6 +334,16 @@
 MEDIA_URL = urljoin(path_prefix, "media/")
 MEDIA_ROOT = os.path.join(conf.KOLIBRI_HOME, "media")
 
+FILE_UPLOAD_TEMP_DIR = os.path.join(conf.KOLIBRI_HOME, "tmp")
+
+if not os.path.exists(FILE_UPLOAD_TEMP_DIR):
+    os.mkdir(FILE_UPLOAD_TEMP_DIR)
+
+
+FILE_UPLOAD_HANDLERS = [
+    "django.core.files.uploadhandler.TemporaryFileUploadHandler",
```

Comment:
Does Django automatically handle the moving out of temporary storage as a part of this? I tried to deduce this from the documentation that was linked (and the "managing files" that is referenced within those pages) and couldn't quite figure it out

Reply:
No - I don't think it does any explicit cleanup.

For context, this change is being made in support of some refactor of this code: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/auth/tasks.py#L48 which already puts things into this folder, and is similarly not cleaning things up, so I don't think this causes any regressions at least.